tests/e2e/expected_results/slsa-verifier/slsa-verifier_yaml_invalid.json

```diff
@@ -1686,7 +1686,7 @@
                 "FAILED": 0,
                 "PASSED": 7,
                 "SKIPPED": 0,
-                "UNKNOWN": 0
+                "UNKNOWN": 1
```

Comment:
I wonder which check was flagged as UNKNOWN in these cases?

Reply:
I think it's the Provenance Expectation check when the expectation file is not valid.